src/locutus/model/terminology.py

```diff
@@ -8,6 +8,8 @@
 
 import pdb
 
+from rich import print

```

Comment:
As much as I love using print from rich, I think we should avoid deploying with it in place since it may make the log look weird due to color (which is going to become just weird strings of text to open and close colored passages). So, if you could remove this, that would be great.

Reply:
Removed print import and pushed the changes.